libraries/bot-azure/src/main/java/com/microsoft/bot/azure/blobs/BlobsStorage.java

```diff
@@ -197,7 +211,17 @@ private static String getBlobName(String key)
             try (ByteArrayOutputStream outputStream = new ByteArrayOutputStream()) {
                 blobReference.download(outputStream);
                 String contentString  = outputStream.toString();
-                Object obj = jacksonAdapter.deserialize(contentString , Object.class);
+
+                Object obj;
+                // We are doing this try/catch because we are receiving String or HashMap
+                try {
+                    // We need to deserialize to an object class since there are contentString which has a String type
+                    obj = objectMapper.readValue(contentString, String.class);
```

Comment:
```suggestion
                    obj = objectMapper.readValue(contentString, Object.class);
```

After checking C#, it will be better to be `Object` instead of specifying to `String`